### ISSUE TITLE
Issue with redirect_to in rails 7 

### ISSUE THREAD

**Issue by danlo** [authrocket/authrocket-ruby#2]
source: https://api.rubyonrails.org/classes/ActionController/Redirecting.html#method-i-redirect_to  ( Open Redirect protection )
article: https://www.mintbit.com/blog/ensure-location-safe-before-redirect-rails7

Greetings,

Rails 7, has an option in the "redirect_to" to prevent redirecting to outside links called:  "allow_other_host".  (See above)

In our specific case our https requests is routed as such

```
                    www.example.tld                           something.something.tld
browser => (load balancer: reverse proxy) => server 
```

The issue that comes back is that the redirect_to used by AuthRocket after a successful signin is to "https://www.example.tld", however, since that is not our domain name in rails in the app (which is something.something.tld), the RAILS code will throw a ActionController::Redirecting::UnsafeRedirectError error.

Recommend solution:

Replace all redirect_to statements with this
```ruby
if Rails.gem_version >= Gem::Version.new('7.0.0')
  redirect_to url, allow_other_host: true
else
  redirect_to url
end
```

Regards,

Daniel

**Comment by zarqman** [comment]
Thanks for bringing this up. We'll update the README and gem shortly.

A couple of potential workarounds in the near-term (for you or anyone else):

1. Add your LoginRocket domain as a known/safe domain:
```ruby
# add to config/environments/<environment>.rb:
config.hosts << 'sample.e2.loginrocket.com' # or your LoginRocket custom domain if using one
# (This may work from application.rb too; need to test it though.)
```

2. If your app already handles redirects safely (which hopefully was already the case if it was on Rails ≤ 6.1 previously), disable the new Rails protection app-wide:
```ruby
# add to application.rb:
config.action_controller.raise_on_open_redirects = false
```

**Comment by zarqman** [comment]
This has been addressed in 2c8380f3d796bc63edb1cc2ae5c1d8e59ab670e3 and may be tested from master:
```ruby
gem 'authrocket', require: 'authrocket/rails', github: 'authrocket/authrocket-ruby'
```

If you get a chance to test it, let us know how it goes.

**Comment by danlo** [comment]
This works for me!
